usecase/src/main/java/org/mobilitydata/gtfsvalidator/usecase/ValidateAllRequiredFilePresence.java

```diff
@@ -52,15 +52,14 @@ public ValidateAllRequiredFilePresence(final GtfsSpecRepository specRepo,
      * A new notice is generated each time a file marked as "required" is missing from a {@link RawFileRepository}
      * instance. This notice is then added to the {@link ValidationResultRepository} provided in the constructor.
      *
-     * @return a list of notices generated each time a file marked as "required" is missing from a {@link RawFileRepository} instance.
+     * @return a list of String containing the name of all files that are present in rawFileRepo
      */
     public ArrayList<String> execute() {
         if (!rawFileRepo.getFilenameAll().containsAll(specRepo.getRequiredFilenameList())) {
-
             specRepo.getRequiredFilenameList().stream()
                     .filter(requiredFile -> !rawFileRepo.getFilenameAll().contains(requiredFile))
                     .forEach(missingFile -> resultRepo.addNotice(new MissingRequiredFileNotice(missingFile)));
         }
-        return new ArrayList<>(specRepo.getRequiredFilenameList());
+        return new ArrayList<>(rawFileRepo.getFilenameAll());
```

Comment:
We'll chat about that when back, @lionel-nj

Reply:
When fixing test, a few questions popped-up. Will discuss them during out net technical meeting @fabrice-v

Reply:
Documentation has been updated for code clarity as well as the confusing variable names. See https://github.com/MobilityData/gtfs-validator/commit/abe16f2dd76a069a62b9b55c3561785db5c9758d